components/feature/webcompat/src/main/java/mozilla/components/feature/webcompat/WebCompatFeature.kt

```diff
@@ -16,8 +16,11 @@ object WebCompatFeature {
     internal const val WEBCOMPAT_EXTENSION_ID = "webcompat@mozilla.com"
     internal const val WEBCOMPAT_EXTENSION_URL = "resource://android/assets/extensions/webcompat/"
 
-    fun install(engine: Engine) {
-        engine.installWebExtension(WEBCOMPAT_EXTENSION_ID, WEBCOMPAT_EXTENSION_URL,
+    /**
+     * Installs the web extension in the runtime through the WebExtensionManager install method
+     */
+    fun install(manager: WebExtensionRuntime) {
```

Comment:
nit: parameter should be called `runtime` now.

Reply:
Updated